Documentation/ldap-connector.md

```diff
@@ -9,25 +9,44 @@ The connector executes two primary queries:
 1. Finding the user based on the end user's credentials.
 2. Searching for groups using the user entry.
 
+## Security considerations
+
+Dex attempts to bind with the backing LDAP server using the end user's _plain text password_. Though some LDAP implementations allow passing hashed passwords, dex doesn't support hashing and instead _strongly recommends that all administrators just use TLS_. This can often be achieved by using port 636 instead of 389, and administrators that choose 389 are actively leaking passwords.
```

Comment:
Yes, you're correct. I think we never worked out the STARTTLS vv LDAPS strategy. Biggest goal of this section was to prevent users from just using port 389 without any encryption. Please feel free to send a follow up or open an issue if you're interested.